avocado/core/tree.py

```diff
@@ -59,7 +59,8 @@ def update(self, items):
                                               for item in items])
 
     def __str__(self):
-        return 'FilterSet([%s])' % ', '.join(["'%s'" % i for i in self])
+        return ('FilterSet([%s])'
```

Comment:
It'd be nice to add a test here: maybe a tweak to the existing fingerprint tests to trigger the behavior we don't want and fix here.